lib/rspec/core/option_parser.rb

```diff
@@ -319,5 +323,10 @@ def configure_only_failures(options)
       options[:only_failures] = true
       add_tag_filter(options, :inclusion_filter, :last_run_status, 'failed')
     end
+
+    def configure_only_pending(options)
+      options[:only_pending] = true
+      add_tag_filter(options, :inclusion_filter, :last_run_status, 'pending')
+    end
```

Comment:
These could probably be collapsed to a `configure_only(options, :pending)` `configure_only(options, :failures)` type call